packages/warriorjs-core/src/Unit.js

```diff
@@ -207,6 +211,18 @@ class Unit {
     this.score += points;
   }
 
+  /**
+   * Substract the given points from the score.
+   *
+   * @param {number} points The points to lose.
+   */
+  losePoints(points) {
+    this.score -= points;
+    if (this.score < 0) {
```

Comment:
@Terseus We should allow the score of the warrior to go under zero, applying the full penalty for killing a captive no matter what was the score at that moment. For example, let's say that the penalty for killing a captive is 20 points, we don't want to subtract less than 20 just because the warrior had only accumulated 15 points when he killed it, we want the final score to be deterministic.

Reply:
I though it would be wrong to set the score under zero, but now you say it making score deterministic is a great point.